lumen/util.py

```diff
@@ -82,3 +85,54 @@ def parse_timedelta(time_str):
         if param:
             time_params[name] = int(param)
     return dt.timedelta(**time_params)
+
+
+def _j_getenv(x):
+    if isinstance(x, Undefined):
+        x = x._undefined_name
+    return os.getenv(x, '')
```

Comment:
Should the default value be something recognizable to a user, e.g. "UNDEFINED_LUMEN_ENV_VARIABLE", so that they can detect when the environment has not been set up properly to run this app?

Reply:
Or maybe "UNDEFINED ENVVAR {x}", so that they can tell what it was looking for?

Reply:
In most cases they would not detect this either. I've mostly followed intake semantics here.